tests/compare_ncfile.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import os
+import sys 
+import re
+import numpy as np
+##from __future__ import print_function
+from netCDF4 import Dataset
+
+nc1 = Dataset(sys.argv[1])
+nc2 = Dataset(sys.argv[2])
+for varname in nc1.variables.keys():
```

Comment:
Shouldn't we first check if the variables in the files are the same?  What if variables in either file are missing or added?